### PR TITLE
created postgresdb for cropcloud data

### PR DESCRIPTION
**Pull request by RAV10K1**
# Description

Replace with description of PR

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [X] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [X] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Comment by RAV10K1** [comment]
Added new hosted PostgreSQL db for Jimmy's cropcloud. Will still need to update the image_url column with actual S3 bucket URLs for image retrieval.

File: app/api/cropcloud_db.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        cur.execute(sql_query, row)
+
+conn.commit()
+
+    
+
+
+
+
+
```
Comment (jrslagle) [inline app/api/cropcloud_db.py]: Do you mind deleting these extra lines?

File: app/utils/img_processing/google_api.py [hunk 2]
```diff
@@ -43,7 +43,7 @@ def __init__(self):
             raise RuntimeError("Missing Google Credentials, Exiting app")
 
         # init the client for use by the functions
-        self.client = vision.ImageAnnotatorClient()
+        self.client = vision_v1.ImageAnnotatorClient()
 
         self.text_moderator = BadWordTextModerator(
             path.join(
```
Comment (jrslagle) [inline app/utils/img_processing/google_api.py:49]: Does this fix the Google Vision API? This seems off topic to setting up a database connection. Maybe put this in a separate PR? (make a new commit that puts this file back the the way it was, create a new branch, paste these changes, commit them to the new branch, and make a PR for it.

**Review by jrslagle — CHANGES_REQUESTED** [review]
There seems to be a file called
"CUserstemsyDocumentsGitHubebtesttest_data.db"
that is empty. What does it do?

File: app/utils/img_processing/google_api.py (hunk 2 above)
Comment (RAV10K1) [reply app/utils/img_processing/google_api.py:49]: When I pulled from the main repo it wouldn't deploy locally and threw an error saying that this module could not be resolved. Upon research it appears Google has made some changes to the vision API and that the types should now be imported directly from google.cloud. I'll revert and make a separate PR for that. Thanks!

File: app/api/cropcloud_db.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+# Getting data from .csv
+with open('37 stories.csv', 'r') as f:
+    reader = csv.reader(f)
+    next(reader)
+    for row in reader:
+        cur.execute(sql_query, row)
+
+conn.commit()
+
```
Comment (TemsyChen) [inline app/api/cropcloud_db.py:46]: Looks great! I would add conn.close() at the end, for good practice.

**Review by TemsyChen — CHANGES_REQUESTED** [review]
Looks great! I would add conn.close() at the end, for good practice.

File: .gitignore [hunk 4]
```diff
@@ -145,3 +145,6 @@ cython_debug/
 .elasticbeanstalk/*
 !.elasticbeanstalk/*.cfg.yml
 !.elasticbeanstalk/*.global.yml
+
+# for test purposes
+*.csv
```
Comment (benjcrich) [inline .gitignore:150]: Should be removed since only for test purposes.

File: app/api/cropcloud_db.py [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+            if row['id'] == id:
+                return row['submission_datetime']
+
+# Tests
+
+db = CropData()
+
+print(db.GetImage(3))
+print(db.GetTranscript(3))
+print(db.GetSubmissionDateTime(3))
```
Comment (benjcrich) [inline app/api/cropcloud_db.py:83]: Add new line
